packages/lucide/.yo-rc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "generator-react-native-vector-icons": {
+    "packageName": "lucide",
+    "postScriptName": "fontcustom",
```

Comment:
the `postScriptName` should be `lucide` based on checking the ttf file.
However you can leave it out as it defaults to the packageName